op-devstack/sysext/l2.go

```diff
@@ -103,6 +105,32 @@ func (o *Orchestrator) hydrateL2ELCL(node *descriptors.Node, l2Net stack.Extensi
 	l2CL.(stack.LinkableL2CLNode).LinkEL(l2EL)
 }
 
+func (o *Orchestrator) hydrateConductors(node *descriptors.Node, l2Net stack.ExtensibleL2Network) {
+	require := l2Net.T().Require()
+	l2ID := l2Net.ID()
+
+	conductorService, ok := node.Services[ConductorServiceName]
+	if !ok {
+		l2Net.Logger().Debug("L2 net node is missing a conductor service", "node", node.Name, "l2", l2ID)
+		return
+	}
+
+	endpoint, _, err := o.findProtocolService(conductorService, HTTPProtocol)
```

Comment:
probably use o.rpcClient?

Reply:
Yeah I really wanted to but the issue is that this client's ultimate destination is `conductorRpc.NewAPIClient(<rpc_client>)` (within the shim) where conductorRpc is the op-conductor package.

It expects the rpc_client to be of the type rpc.Client (returned by rpc.DialContext) than client.Rpc (returned by o.rpcClient).

Hence, had to fall back on the former.

Also, I couldn't find a way to infer the rpc.Client from a client.Rpc object, otherwise I would have fed client.Rpc to the shim and shim could have translated the rpc.Client out of it.

Reply:
Although, op-conductor package behind the scenes, just makes use of only CallContext from this rpc_client, which can very well be dealt from client.Rpc type (same as the one returned by o.rpcClient) but I didn't necessarily want to introduce more intrusive changes to this PR.